apiary.apib

```diff
@@ -1256,6 +1262,21 @@ Delete an existing invoice. Only possible for draft invoices or the last booked
 
 + Response 204
 
+### invoices.registerPayment [POST /invoices.registerPayment]
+
+Register a payment for an invoice.
+
++ Request (application/json;charset=utf-8)
+
+    + Attributes (object)
+        + id: `7abb325c-e063-42a4-8fb4-1b730759645a` (string, required)
```

Comment:
I thought we discussed we would register payments using a reference rather than introducing payment uuids?

Reply:
I thought we were going to use the uuid, will bring up the discussion in the channel

Reply:
btw, this `id` is the invoice id :p

 :ok: by @stivni on 01/02/2018 17:03:54